index.rb

```diff
@@ -12,14 +10,22 @@ def decode_char (char)
     '--..' => 'Z', ' ' => ' '
   }
 
-  morseCode[char]
+  morse_code[char]
+end
+
+def decode_word(string)
+  spliter = ' '
+  string.split(spliter).map { |char| decode_char(char) }.join
 end
 
+# decode_char(".-")
+
+def decode(message)
+  message.split('   ').map { |string| decode_word(string) }.join
+end
```

Comment:
- [OPTIONAL] I think it would be better to use `join('  ')` to make a space between each word 💯